crates/zkforge/bin/cmd/test/mod.rs

```diff
@@ -151,6 +155,7 @@ impl TestArgs {
         let mut project = config.project()?;
         // load the zkSolc config
         let mut zksolc_cfg = config.zk_solc_config().map_err(|e| eyre::eyre!(e))?;
+        zksolc_cfg.contracts_to_compile = self.contracts_to_compile.clone();
```

Comment:
Shouldnt this be for builds, and scripts as well?

Reply:
Yes you are right sir, this should be replicated in build and scripts as well. Will do now 👍